CHANGELOG.md

```diff
@@ -6,7 +6,9 @@
 
 ### Breaking Changes
 
-_None_
+* Strings: due to the bugfix for fallback translations, custom `lookupFunction`s will need to accept 3 arguments (table, key and value), up from 2 arguments (table and key).    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/230aa4f3c16364cb2a8fdb33569c9b0810f49805/CHANGELOG.md#L9"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>